lib/openscaptest.pm

```diff
@@ -70,7 +70,7 @@ sub validate_result {
         assert_script_run "xmllint --noout $xml_args $result_file";
     }
 
-    validate_script_output "cat $result_file", sub { $match };
+    validate_script_output "cat $result_file", sub { $match }, timeout => 300;
```

Comment:
LGTM.
Just a question, if it also works like this? 
**validate_script_output ("cat $result_file", sub { $match }, timeout => 300);**

Reply:
> LGTM.
> Just a question, if it also works like this?
> **validate_script_output ("cat $result_file", sub { $match }, timeout => 300);**

yes.